app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -363,6 +364,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         settings().shouldReturnToBrowser =
             components.core.store.state.getNormalOrPrivateTabs(private = false).isNotEmpty()
 
+        Metrics.inactiveTabsCount.set(components.core.store.state.actualInactiveTabs(settings()).size.toLong())
```

Comment:
Added the probe here looking for the best compromise between 
"not to soon" - FenixApplication#onCreate + "don't impact performance" and 
"not too low so that this will be called more often than needed"
and because I saw the `settings()...` calls just below this which store the right values in SharedPreferences to make them available for the metrics ping. (though about using a similar mechanism to store this value until it's needed for the ping but it seemed superfluous if Glean already does this for probes with `lifetime: Ping`)
Open to suggestions to move this elsewhere.

Reply:
In metrics.yml it just says "How many inactive tabs does the user have." - but that value changes over time. Measuring here vs. measuring it on app start may result in completely different values? 🤔 Unfortunately the spreadsheet also doesn't mention *when* this is getting measured. Does this need to be clarified?

Just wanted to mention, if you need it on app start and want to avoid the race. There's a `restoreComplete` boolean in the state, that you can wait for. I think we use this already to go back to home if after restore there are no tabs anymore (e.g. we purged inactive ones).

Reply:
Ohh, the `restoreComplete` seems nice, thank you!

I initially added this in FenixApplication#setStartupMetrics since I saw there similar telemetry for the number of open tabs , topSites and thoughout that since we only send the last count of inactive tab once per day to Glean this place is a good fit.
Thinking now about the granularity of inactive tabs updates maybe a more appropriate place would be the tabs tray - from where the event that also contains this count as extra key is logged now. This would cover the scenario in which the user closes some inactive tab, then the app and will only open it again when the `metrics` ping is scheduled for upload thus potentially loosing the update from the previous day (not entirely lost since it could be reported in the next ping).

Will move this to tabs tray, update documentation and check with Kimmy.